cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -314,14 +325,63 @@ func TestEvaluateExecute(t *testing.T) {
 				},
 
 				ExpectedOutputValidate: func(t *testing.T, output, resultPath string) {
-					assert.Contains(t, output, "skipping unavailable provider \"openrouter\"")
+					assert.Contains(t, output, "Skipping unavailable provider \"openrouter\"")
 				},
 				ExpectedResultFiles: map[string]func(t *testing.T, filePath string, data string){
 					"evaluation.log": nil,
 				},
-				ExpectedPanic: "ERROR: model openrouter/auto does not exist. Valid models are: symflower/symbolic-execution",
+				ExpectedPanicContains: "ERROR: model openrouter/auto does not exist",
 			})
 		})
+		t.Run("Ollama", func(t *testing.T) {
+			if !osutil.IsLinux() {
+				t.Skipf("Installation of Ollama is not supported on this OS")
+			}
+
+			{
+				var shutdown func() (err error)
+				defer func() {
+					if shutdown != nil {
+						require.NoError(t, shutdown())
+					}
+				}()
+				validate(t, &testCase{
+					Name: "Pulled Model",
+
+					Before: func(t *testing.T, logger *log.Logger, resultPath string) {
+						var err error
+						shutdown, err = tools.OllamaStart(logger, tools.OllamaPath, tools.OllamaURL)
+						require.NoError(t, err)
+
+						require.NoError(t, tools.OllamaPull(logger, tools.OllamaPath, tools.OllamaURL, "qwen:0.5b"))
+					},
+					After: func(t *testing.T, logger *log.Logger, resultPath string) {
+						require.NoError(t, shutdown())
+						shutdown = nil
+					},
```

Comment:
Could probably get rid of this shutdown and rely on the `defer` one outside of the `valdidate`... Not really a fan of doubling this up.